app/policies/event_policy.rb

```diff
@@ -1,11 +1,19 @@
-
-
 class EventPolicy < ApplicationPolicy
-  attr_reader :user, :event
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
 
-  def initialize(user, event)
-    @user = user
-    @event = event
+    def resolve
```

Comment:
Should users be able to see non_private events that they have registered for?

Reply:
I think I did it now.